data/campaigns/fri05.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,371 @@
+}
+
+intro_6 = {
+   title = _("Greetings"),
+   body=reebaud(_("Hospitality"),
+      -- TRANSLATORS: Reebaud – Introduction 6
+      _([[Peace and thanks for the friendly greeting to you, Ocke Ketelsen! It is a pleasant change not to be forced to go to war against someone on whose island we land. Yet your welcome sounds oddly short. Since your words are so warm, I can only conclude that some other sort of trouble has befallen you. Assuming you don’t mind if I ask and that this question does not appear impertinent, I would like to know if we can help you in any way.]]))
+}
+
+intro_7 = {
+   title = _("Greetings"),
+   body=ketelsen(_("Welcome to Helgoland!"),
+      -- TRANSLATORS: Ocke Ketelsen – Introduction 7
+      _([[You are remarkably astute. Indeed we have a problem, and you can see it right over there.]]))
+}
+
+intro_8 = {
+   title = _("Greetings"),
+   body=henneke(_("Enemies"),
+      -- TRANSLATORS: Henneke Lembeck – Introduction 8
+      _([[Danes! Does Atterdag’s thirst for conquest reach this far then?]]))
+}
+
+intro_9 = {
+   title = _("Greetings"),
+   body=ketelsen(_("Enemies"),
+      -- TRANSLATORS: Ocke Ketelsen – Introduction 9
+      _([[It does. I see that you recognize this enemy, and I would be interested in hearing how you met – though I imagine it will not have gone well for them!]]))
+}
+
+intro_10 = {
+   title = _("Greetings"),
+   body=claus(_("Enemies"),
+      -- TRANSLATORS: Claus Lembeck – Introduction 10
+      _([[This assumption is untrue, to my great chagrin. For the sake of brevity, I will only say that Atterdag invaded Föhr with such a mighty force that I was compelled to leave the island and ask Reebaud to help me reclaim it. In fact, we were sailing to Föhr to drive the invaders out once and for all when a storm blew us off our course and to Helgoland.]])
+      .. paragraphdivider() ..
+      -- TRANSLATORS: Claus Lembeck – Introduction 10
+      _([[That it did so might even be heavenly intervention. We will gladly help you fight this enemy. A warm-up battle, so to speak – I am looking forward to it!]]))
+      .. new_objectives(obj_fight),
+   h = objective_message_height,
+}
+
+trade_1 = {
+   title = _("War Preparations"),
+   body=ketelsen(_("Enemies"),
+      -- TRANSLATORS: Ocke Ketelsen – Trade 1
+      _([[Your enthusiasm warms me as much as your plight engages my sympathy. But the enemies are more powerful than they may appear – they have brought an enormous force of Danish soldiers, and even your mighty soldiers will not be able to defeat them all.]])
+      .. paragraphdivider() ..
+      -- TRANSLATORS: Ocke Ketelsen – Trade 1
+      _([[Our island is not accustomed to this kind of warfare. We are fishers and farmers, we do not have the infrastructure to support a drawn-out battle. Your expertise in this area would be much appreciated.]])
```

Comment:
"Infrastructure" sounds anachronistic to me here.

(though we have such anachronisms in other campaigns as well, but I think they should be corrected there too, just haven't got round to start working on it)